python/paddle/fluid/transpiler/inference_transpiler.py

```diff
@@ -54,19 +55,57 @@ def transpile(self, program, place, scope=None):
         if not isinstance(scope, core.Scope):
             raise TypeError("scope should be as Scope type or None")
         self.fuse_batch_norm(program, place, scope)
+        use_mkldnn = bool(os.getenv("FLAGS_use_mkldnn", False))
+        if use_mkldnn:
+            self.fuse_relu(program)
+
+    def fuse_relu(self, program):
+        '''
+        Transpile the program by fused relu activation for MKLDNN program.
+
+        Relu activation following batch norm OP can be fused by adding
+        'fuse_with_relu' attribute to batch norm OP.
+
+        The result of fuse is:
+            - before:
+                - batch_norm->relu->any_other_op
+            - after:
+                - batch_norm->any_other_op
```

Comment:
The format of line 69-73 is not correct. You can use https://github.com/PaddlePaddle/FluidDoc to see the generated html. And paste the generated picture here like https://github.com/PaddlePaddle/Paddle/pull/11521.

If you have any question about how to generate the API reference, please feel open to ask me.